training-react/tic-tac-toe/src/app/screens/ErrorPage/styles.scss

```diff
@@ -0,0 +1,28 @@
+@import 'variables/colors';
+
+.container-pag-error {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  width: 100vw;
+
+  .msg-error {
+    align-items: center;
+    background-color: $concrete;
+    border-radius: 5px;
+    display: flex;
+    flex-wrap: wrap;
+    height: 140px;
+    justify-content: center;
+    padding: 10px 20px;
+    text-align: center;
```

Comment:
Delete this property when you use `flex-box` this property doesn't work as well as you're use `justify-content` property